dev/src/main/resources/messages/ui/CodewindUIBundle.properties

```diff
@@ -216,6 +216,8 @@ ShowLogFilesMenu=&Log Files
 ShowAllLogFilesAction=&Show All
 CloseAllLogFilesAction=&Close All
 LogFilesToolWindow=Log Files
+NoLogFilesAvailableMessage=The project {0} does not have any logs available at this time. Wait for the project to build, and try again.
+NoLogFilesAvailableTitle=No Log Files Available
```

Comment:
I'm seeing a mix of titles capitalized like book titles and titles with only the first word capitalized. It would be good to be consistent if we can. :)

Reply:
The IntelliJ dialog/message titles are like this.  I'll try to keep it consistent.  So the NoLogFilesAvailableTitle is like "Open Project" shown below:

<img width="502" alt="Screen Shot 2020-04-16 at 10 40 27 AM" src="https://user-images.githubusercontent.com/7726022/79469887-bbdc7800-7fce-11ea-9cf6-70ef9f0fcaae.png">